package-lock.json

```diff
@@ -9307,6 +9307,7 @@
       "resolved": "https://registry.npmjs.org/bufferutil/-/bufferutil-4.0.5.tgz",
       "integrity": "sha512-HTm14iMQKK2FjFLRTM5lAVcyaUzOnqbPtesFIvREgXpJHdQm8bWS+GkQgIkfaBYRHuCnea7w8UVNfwiAQhlr9A==",
       "dev": true,
+      "hasInstallScript": true,
```

Comment:
I was wondering how come we have these changes?

Reply:
The package-lock.json was mentioned as a suspect when we met to discuss the failing test in CI, so I included it.  The `hasInstallScript` indicates that a package has an install script to run when installing.